<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_Form</class>
 <widget class="QWidget" name="User_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>561</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(126,190,247)</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_goods">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>230</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_goods">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>读取的商品编号:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_available_COM">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Open_Card">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>741</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开读卡器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_add_goods">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>741</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>添加商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_SC">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>741</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>结账确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Scan">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>扫描读卡器</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>821</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240,10,10)</string>
     </property>
     <property name="text">
      <string>提示:请扫描商品电子标签</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_delete_item">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>470</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>删除选中商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_delete_items">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>470</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>删除全部商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_Settle">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>470</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>结算</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tab_goods">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>821</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
